README.md

```diff
@@ -73,6 +73,12 @@ docker run -d \
 
 ```
 
+### Deploy on Sealos
+
+[Sealos](https://github.com/labring/sealos) is a cloud operating system distribution based on the Kubernetes kernel. It enables the initialization of all application dependencies, including the database, with a single click.
+
+[![](https://raw.githubusercontent.com/labring-actions/templates/main/Deploy-on-Sealos.svg)](https://template.cloud.sealos.io/deploy?templateName=palworld)
+
```

Comment:
My only nit-pick with this is that it's adding what's effectively an ad to the readme

Instead, could a basic hyper link to the SealOS template be added under the Kubernetes section?